hal/hal_i2c.h

```diff
@@ -65,7 +77,8 @@ void i2c_set_slave_mode(const i2c_periph_inst_t* I2C_instance, const unsigned sh
  * @param stop_bit Does this transaction end with or without a stop-bit: Value 1 is with stop-bit
  *                                                                       Value 0 is without stop-bit
  */
-void i2c_write_blocking(const i2c_periph_inst_t* I2C_instance, const unsigned char addr, const unsigned char* write_buff, const unsigned char size, bool stop_bit);
+void i2c_write_blocking(const i2c_periph_inst_t* I2C_instance, const unsigned char addr,
+                        const unsigned char* write_buff, const unsigned char size, bool stop_bit);
 
```

Comment:
I think this should be fine, alternative could be to make an Enum for such a constructions, but will add some overhead which might be a reason to keep it as is.